src/shared/pins.ts

```diff
@@ -100,7 +100,8 @@ export const getPins = async (targetVersion: string): Promise<Pins> => {
   const tag = targetVersion.startsWith('v') ? targetVersion : `v${targetVersion}`;
   for (const url of [
     `https://raw.githubusercontent.com/invoke-ai/InvokeAI/${tag}/pins.json`,
-    `https://fastly.jsdelivr.net/gh/invoke-ai/InvokeAI@${tag}/pins.json`
+    `https://cdn.jsdelivr.net/gh/invoke-ai/InvokeAI@${tag}/pins.json`,
+    `https://fastly.jsdelivr.net/gh/invoke-ai/InvokeAI@${tag}/pins.json`,
```

Comment:
Using the `fastly` subdomain bypasses jsdelivr's load balancing and routing logic. We really shouldn't be using it.